docs/manual/src/tutorial/Rust_scaffolding.md

```diff
@@ -30,7 +35,16 @@ uniffi::include_scaffolding!("math");
 
 **Note:** The file name is always `<namespace>.uniffi.rs`.
 
-Great! `add` is ready to see the outside world!
+### Setup for crates using only proc macros
+
+If you are only using proc macros, you can skip `build.rs` entirely!
+All you need to do is add this to the top of `lib.rs`:
+
+```rust
+uniffi::setup_scaffolding!();
```

Comment:
please add something like: The setup_scaffolding function takes one optional parameter, the ["namespace"](../udl/namespace,md] used by the component. If not specified, the name of the crate will be used as the namespace.

Reply:
Good call; that's super helpful.